content/writing_tests/migration-sakuli-1.md

```diff
@@ -0,0 +1,168 @@
+---
+title : "Migration from Sakuli 1"
+date :  2020-05-20T08:56:58+02:00
+weight : 10
```

Comment:
What's the reason for `weight: 10`?

I also noticed the ordering of posts is off by one and starts at `2`

Reply:
Just to ensure it stays at the bottom :smile:

Reply:
Altered the weight to 8